x-pack/platform/plugins/shared/lens/server/api/routes/visualizations/create.ts

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { schema } from '@kbn/config-schema';
+
+import { CONTENT_ID } from '../../../../common/content_management';
+import {
+  PUBLIC_API_PATH,
+  PUBLIC_API_VERSION,
+  PUBLIC_API_CONTENT_MANAGEMENT_VERSION,
+  PUBLIC_API_ACCESS,
+} from '../../constants';
+import {
+  lensAttributesSchema,
+  lensCreateOptionsSchema,
+  lensCreateResultSchema,
+} from '../../../content_management/v1';
+import { RegisterAPIRouteFn } from '../../types';
+
+export const registerLensVisualizationsCreateAPIRoute: RegisterAPIRouteFn = (
+  router,
+  { contentManagement }
+) => {
+  const createRoute = router.post({
+    path: `${PUBLIC_API_PATH}/visualizations`,
```

Comment:
I used this as `lens/visualizations/` instead of `lens/` to be more explicit about the resource being change and in case there are other resources to add later.